docs/organizing-and-exploring-results-in-the-ui/index.rst

```diff
@@ -3,14 +3,36 @@ Working with Neptune UI
 
 |Youtube Video dashboard|
 
-**More info:**
+Neptune UI has 3 major sections: Experiments, Notebooks and Wiki.
 
-.. toctree::
-   :maxdepth: 2
+**Experiments** section is the place where you can:
+
+- organize experiments into groups and views
+- filter and search experiments
+- compare experiments
+- drill down to experiment details like charts and code
```

Comment:
You can also watch running exp.

Consider adding info that everything is shearable in here.